.github/dockerfiles/ov_build/ubuntu_20_04_x64/Dockerfile

```diff
@@ -22,13 +22,10 @@ RUN apt-get update && \
         ca-certificates \
         gpg-agent \
         tzdata \
-        # Pythons
-        python3.8-dev \
-        python3.8-venv \
-        python3.8-distutils \
-        python3.11-dev \
-        python3.11-venv \
-        python3.11-distutils \
+        # Python
+        python3-dev \
```

Comment:
will be installed 3.11 version by default? 
as far as i remember we need 3.8 to build dep packages, where we will build them now?

Reply:
never mind, i see that python 3.8 is default python version on ubuntu 20.